httpx/models.py

```diff
@@ -845,6 +846,19 @@ def cookies(self) -> "Cookies":
             self._cookies.extract_cookies(self)
         return self._cookies
 
+    @property
+    def links(
+        self
+    ) -> typing.Dict[typing.Optional[typing.Any], typing.Dict[str, typing.Any]]:
+        header = self.headers.get("link")
+        ldict = {}
+        if header:
+            links = parse_header_links(header)
+            for link in links:
+                key = link.get("rel") or link.get("url")
```

Comment:
We need some unit tests for this logic as well.

Reply:
done